migrations/sql/V1.35__add_user_table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE minespace_users (
```

Comment:
Is the `minespace_` prefix meant to denote a higher-order 'type of table'?  @kristimeredith  how do you feel about `mds_users` to denote that it's "application wide"?

Reply:
@garywong-bc did you mean to tag me in this?
If so, yes, mds_user would be a good name... not plural ;)

Reply:
Users in the table will come from the minespace (public facing) website, (Currently a different front end for the existing backend and db). The long term vision is for minespace to be freestanding application where this table would be transplanted into that application's db. mds_users would be inspectors and other EMPR staff. That's why the prefix minespace, is being used.